lighttpd_rust_amalgamated/src/lighttpd_amalgamated.rs

```diff
@@ -1,8 +1,17 @@
 #![allow(dead_code, mutable_transmutes, non_camel_case_types, non_snake_case, non_upper_case_globals, unused_assignments, unused_mut)]
 #![feature(c_variadic, core_intrinsics, extern_types, label_break_value)]
-#[macro_use]
-extern crate c2rust_bitfields;
-use ::lighttpd_rust_amalgamated::*;
+#![allow(dead_code)]
+#![allow(mutable_transmutes)]
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
+#![allow(non_upper_case_globals)]
+#![allow(unused_assignments)]
+#![allow(unused_mut)]
+
+#[macro_use] extern crate libc;
+#[macro_use] extern crate c2rust_bitfields;
```

Comment:
I think only `c2rust_bitfields` needs `#[macro_use]` since it's the one with macros.

Reply:
9487609